include/lbann/layers/regularizers/dropout.hpp

```diff
@@ -201,7 +168,7 @@ class dropout : public regularizer_layer {
     // Do nothing if dropout is disabled
     const auto& mode = this->m_model->get_execution_mode();
     if (mode != execution_mode::training || m_keep_prob < EvalType(0)) {
-      El::LockedView(output, input);
+      El::Copy(input, output);
```

Comment:
So just to clarify, you are reverting this back to a copy to simplify implementation despite a small performance loss.  Is that correct?